website/docs/docs/dbt-versions/release-notes.md

```diff
@@ -20,6 +20,12 @@ Release notes are grouped by month for both multi-tenant and virtual private clo
 
 ## September 2024
 
+- **New**: You can set descriptions in your YAML files to document [singular data tests](/docs/build/data-tests#singular-tests).
```

Comment:
is this link right? it's not linking to anything that's 'singular-tests' in the data-test page. is it meant to link to 'singular-data-tests'?

Reply:
I think we had to roll this back - so we may want to hold off on adding this to the release notes